ProjectLibertyLabs/gateway
# fix: change end-of-chain scan log message to debug instead of error

# Problem

"Latest finalized block encountered" error messages spam the log.

# Solution

Changed the log level to `debug`.
Went back & forth between this & `verbose`; ultimately decided that even at the verbose level it's too spammy.
Different opinions welcome in the comments section.

## Steps to Verify:

1. Run any gateway service with `DEBUG=true` in the environment
2. Let the scan run & watch the log
3. Logs about the scan reaching end-of-chain should be logged at the `[DEBUG]` level

Closes #643
